org.mwc.cmap.legacy/src/MWC/GUI/Properties/PlainPropertyEditor.java

```diff
@@ -1156,7 +1157,7 @@ private void update(final PropertyDescriptor pd, final Object data)
     // current one for this object - or if there isn't a current value
     if ((newVal == null) || (!newVal.equals(oldVal)))
     {
-      if (newVal == null && pe instanceof SwingDatePropertyEditor)
+      if (newVal == null && newVal instanceof HiResDate)

```

Comment:
@IanMayo . I _think_ you meant here `DatePropertyEditor` . I have pushed it in the next commit.

Reply:
No, IMHO we can test the type of the object being edited, which is more direct than checkin the type of editor in place.

But, I may be wrong...

Reply:
@saulhidalgoaular  was right all along :-)  we've already established that newVal is null, so we know we can't do type checking on it.